### PR TITLE
Ember2 Upgrade - The Assman Cometh

### PR DESCRIPTION
**Pull request by thescubageek**
* Fixes double tap bug on asset uploader, where duplicates were being uploaded on subsequent uploads
* Fixes infinite throbber bug
* Disables uploader while uploading
* Fixes incorrect counter bug

PT: https://www.pivotaltracker.com/story/show/130565721
CMS: https://github.com/g5search/g5-content-management-system/pull/2153

### CODE REVIEW

File: app/templates/components/asset-uploader.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="file-field input-field">
-  <div class="btn upload-files">
+  <div class="btn upload-files {{uploadDisabled}}">
```
Comment (jcstringer) [inline app/templates/components/asset-uploader.hbs]: Nice. You can also do it like this as I have learned and avoid having to code anything in the actual js
```
<div class="btn upload-files{{if disabled ' disabled='disabled' ''}}>
```

**Comment by jcstringer** [comment]
🚢  when 🍏

File: app/templates/components/asset-uploader.hbs (hunk 1 above)
Comment (thescubageek) [inline app/templates/components/asset-uploader.hbs]: Ember2 is so much sexier than it ugly little sister

Comment (thescubageek) [inline app/templates/components/asset-uploader.hbs]: Though we actually want the class itself to be `disabled` -- Materialize takes care of toggling the properties

**Pull request merged**